src/ModalContent/ModalContent.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { dcnb } from 'cnbuilder';
+import PropTypes from 'prop-types';
+import { DismissButton } from '../DismissButton/DismissButton';
+import { Title } from './ModalContent.Title';
+import { Body } from './ModalContent.Body';
+import { Cta } from './ModalContent.Cta';
+import { Footer } from './ModalContent.Footer';
+
+/**
+ * ModalContent Component
+ *
+ * HTML ModalContent element
+ */
+
+export const ModalContentRoot = ({
+  className,
+  children,
+  onClose,
+  ...props
+}) => (
+  <div
+    className={dcnb(
+      'su-bg-white su-max-w-[400px] md:su-max-w-[683px] su-flex su-flex-col su-rs-pt-2 su-rs-pr-3 su-rs-pb-4 su-rs-pl-5',
+      className
+    )}
+    {...props}
+  >
+    <DismissButton
+      id="closeModal"
```

Comment:
🤔 Will this be at risk of being duplicated if there is more than one modal on the page?